specification/batch/resource-manager/Microsoft.Batch/stable/2022-01-01/BatchManagement.json

```diff
@@ -0,0 +1,6019 @@
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Contains information about a certificate."
+    },
+    "ListCertificatesResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Certificate"
+          },
+          "description": "The collection of returned certificates."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The continuation token."
+        }
+      },
+      "description": "Values returned by the List operation."
+    },
+    "DeleteCertificateError": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for display in a user interface."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error. For example, the name of the property in error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeleteCertificateError"
+          },
+          "description": "A list of additional details about the error."
+        }
+      },
+      "required": [
+        "code",
+        "message"
+      ],
+      "description": "An error response from the Batch service."
+    },
+    "DetectorListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DetectorResponse"
+          },
+          "description": "The collection of Batch account detectors returned by the listing operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      },
+      "description": "Values returned by the List operation."
+    },
+    "DetectorResponse": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DetectorResponseProperties",
+          "description": "The properties associated with the detector."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Contains the information for a detector."
+    },
+    "DetectorResponseProperties": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "A base64 encoded string that represents the content of a detector."
```

Comment:
why does this need to be base64 encoded instead of representing the content via a json schema?  How will a user consume this?

Reply:
This is meant to be consumed via the portal. The base64 string is part of [an external API](https://applensdiagnostics.azurewebsites.net/?p=414) that the portal will consume.